data/partners.yaml

```diff
@@ -1,3 +1,6 @@
+- title: arksec
+  logo: arksec-logo.png
+  link: https://arksec.cn/
```

Comment:
@simonsuns please move it on second place as we try to keep everything in alphabetical order! Thank you!

Reply:
It has been fixed